Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -87,14 +87,20 @@ void IOnClickListener.OnClick(AView v)
 
 		public override bool OnInterceptTouchEvent(MotionEvent ev)
 		{
-			if (Element.InputTransparent && Element.IsEnabled)
-				return false;
+			if (!Element.IsEnabled || Element.InputTransparent)
+				return true;
```

Comment:
Touches seem to be put on a stack from parent to inner most child and this stack is emptied out from that child back to parent. Makes sense. Here, the parent says consume the event and do not send it to children if I'm disabled or input transparent.